comp/core/tagger/collectors/workloadmeta_main.go

```diff
@@ -76,6 +79,24 @@ func (c *WorkloadMetaCollector) initPodMetaAsTags(labelsAsTags, annotationsAsTag
 	c.nsLabelsAsTags, c.globNsLabels = utils.InitMetadataAsTags(nsLabelsAsTags)
 }
 
+func (c *WorkloadMetaCollector) injectHostTags() {
+	duration := config.Datadog.GetDuration("expected_tags_duration")
+	if duration <= 0 {
```

Comment:
Maybe `<= time.Minute`.

Actually, maybe two separate if, the early return one you have already and a following one testing for `<= time.Minute` to throw a warning + early return?

Reply:
I think the warning is a good idea - ill add that. 
We may not need to early return though. Since the timer only fires once per min, if the timeout is < `1m`, we are still bounded by the timers tick interval. So if a user sets this to `10s`, it will still take `~1m` to expire the tags. WDYT?

Reply:
Got it, sounds good!